seqr/views/utils/export_utils.py

```diff
@@ -75,10 +75,9 @@ def _format_files_content(files,  file_format='csv', add_header_prefix=False, bl
             header_display = ['{}-{}'.format(str(header_tuple[0]).zfill(2), header_tuple[1]) for header_tuple in
                               enumerate(header)]
             header_display[0] = header[0]
-        content = DELIMITERS[file_format].join(header_display) + '\n'
         content_rows = [[row.get(key) or blank_value for key in header] for row in rows]
-        content += '\n'.join([
-            DELIMITERS[file_format].join(row) for row in content_rows
+        content = '\n'.join([
```

Comment:
fixes a bug when the table is empty

Reply:
Add a test for an empty table?

Reply:
out of scope, I just happened to notice it here but this endpoint can't return empty tables